engines/mohawk/dialogs.cpp

```diff
@@ -119,6 +120,18 @@ MystOptionsWidget::MystOptionsWidget(GuiObject *boss, const Common::String &name
 		                                             _("The Myst fly by movie was not played by the original engine."));
 	}
 
+	if (!isDemo) {
+		/* 
+		I18N: 
+		This Option is for hard-of-hearing.
+		It makes it easier to solve the spaceship puzzle.
+		Normally game uses strict binary logic here.
+		We change it to use fuzzy logic.
+		By default the option is off.
+		*/
```

Comment:
Multi-line comments have this format:
/**
 * This
 * is
 * a
 * multiline
 * comment
 */

Reply:
Done